frontend/src/AppRouter.tsx

```diff
@@ -233,6 +234,12 @@ const AppRouter: React.FC = () => {
           component={TestFileManagement}
           layout={AuthLayout}
         />
+        <AppRoute
+          path="/test/testDocument"
```

Comment:
if possible we should merge these two path="/testFileManagement"

Reply:
Moved to this one